docs/dev/development.md

```diff
@@ -1,70 +1,90 @@
 # Local Development Setup
-To use this development environment on OSX, you need to have Docker and Boot2docker installed.
+To use this development environment on OSX, you need to have Java, Docker and VirtualBox installed (Boot2docker is optional).
```

Comment:
Virtualbox is not necessary, because by default deployer uses xhyve as hypervisor. `cbd machine create` command initialize an xhyve instance.